configure.sh

```diff
@@ -192,9 +201,3 @@ build --action_env ANDROID_SDK_API_LEVEL="29"
 build --action_env ANDROID_SDK_HOME="/tmp/lce_android"
 
 EOM
-
-if is_windows; then
-  write_to_bazelrc "build --config=v2 --config=windows"
-else
-  write_to_bazelrc "build --config=v2 --config=noaws --config=nogcp --config=nohdfs --config=nonccl"
```

Comment:
I think we should still add `--config=noaws --config=nogcp --config=nohdfs --config=nonccl` to the `build:linux` and `build:macos` definitions since it will slightly speed up compile time by disabling features we don't need.

Reply:
Do those config options not work on Windows?

Reply:
Yeah for some reason I got a conflict between `--config=windows` and `--config=noaws` in one of the TensorFlow internal build rules, I didn't spent time debugging the root cause, but removing removing them from the windows build fixed it.